acl.n3

```diff
@@ -14,7 +14,7 @@
 
 <https://www.w3.org/wiki/WebAccessControl> dc:describes <>.
 <https://github.com/solid/web-access-control-spec> dc:describes <>.
-
+<https://solidproject.org/TR/wac> dc:describes <>.
```

Comment:
It doesn't seem that this property is defined there: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#section-3

Reply:
Good catch! I suspect that `dc:description` was intended in the previous lines and I followed without thinking.

While we are looking at this, would it be appropriate to change the namespace from `/elements/1.1/` to `/terms/` since "DCMI gently encourages use of the `/terms/` namespace"? The ACL Ontology currently only uses `:title` and `:describes` (and we are about to change the latter to `:description` unless another property is preferred.)

I think we can remove statement with https://github.com/solid/web-access-control-spec and keep only https://www.w3.org/wiki/WebAccessControl and https://solidproject.org/TR/wac .

What say you all?

Reply:
Agree we should us a term which exists.  Agree we shoiuld use DCT not DC.  I Like having a pointer to the spec from the ontolgy

Reply:
Updated in https://github.com/solid/vocab/pull/80/commits/f2eaf6a89c28d640226209365f65b0c7b864e527